internal/commands/import.go

```diff
@@ -40,11 +41,34 @@ func runImportCommand(wrapper wrappers.UploadsWrapper) func(cmd *cobra.Command,
 		if err != nil {
 			return err
 		}
+		projectName, err := cmd.Flags().GetString(commonParams.ProjectName)
```

Comment:
should we always have a project in this command? it seems like we had different functonaility

Reply:
import is the new command and project name must be provided